cdp_scrapers/legistar_utils.py

```diff
@@ -99,6 +99,8 @@
 
 LEGISTAR_EV_ITEMS = "EventItems"
 LEGISTAR_EV_ATTACHMENTS = "EventItemMatterAttachments"
+LEGISTAR_EV_MATTER_ID = "EventItemMatterId"
```

Comment:
Just for clarification, what's the difference between `LEGISTAR_EV_MATTER_ID` and `LEGISTAR_MATTER_EXT_ID`?

Reply:
Given the same Legistar Matter, the _value_ will be the same.

`LEGISTAR_EV_MATTER_ID` = "EventItemMatterId" is a field in the Legistar Event Item. So, after querying for a Legistar Event Item, we can then query for the associated Matter using that field.

`LEGISTAR_MATTER_EXT_ID` = "MatterId" is a field in the Legistar Matter. After obtaining a Legistar Matter, we use that field to set `Matter.external_source_id`.